Makefile

```diff
@@ -135,9 +135,11 @@ wheels:
 
 
 download-wheels:
-	./scripts/download_wheels.py $(PANDAS_VERSION)
+	python ./scripts/download_wheels.py $(PANDAS_VERSION)
 	# TODO: Fetch from https://www.lfd.uci.edu/~gohlke/pythonlibs/
 
 
 upload-pypi:
 	twine upload pandas/dist/pandas-$(PANDAS_VERSION)*.{whl,tar.gz} --skip-existing
+
+# twine upload pandas/dist/pandas-1.0.4*.{whl,tar.gz} --skip-existing
```

Comment:
the regex pattern didn't work (can't remember whether was using git bash or Anaconda prompt. Now that using Docker can revert.